src/components/object-counter/object-counter.scss

```diff
@@ -0,0 +1,45 @@
+.field-container {
```

Comment:
You need to wrap everything to .object-counter-container in order to apply all this scss only in this component. We have same classes such as '.field-container' in other components.